### ISSUE TITLE
object detection loss NaN

### ISSUE THREAD

**Issue by waterbearbee** [rgeirhos/texture-vs-shape#4]
Hi Robert Geirhos,

I am currently creating an experiment of object detection with your pertained model. But unfortunately, the loss becomes NaN at the beginning.  Then I reread your paper and find that you apply input whitening for image processing.  So, I really want to know the reason and the impact of this processing.

Kind Regards!

**Comment by michaelisc** [comment]
The [object detection implementation we use](https://github.com/jwyang/faster-rcnn.pytorch) preprocesses images by subtracting the mean intensity of each color channel (in `lib/model/utils/blob.py`):
```
im -= pixel_means
```


In contrast the [PyTorch ImageNet training](https://github.com/pytorch/examples/tree/master/imagenet) we used to train our ImageNet models preprocesses images by normalizing the pixel values to [0,1] followed by channel wise subtraction of the mean and division by the standard deviation. Thus we changed our object detection preprocessing to follow the same procedure:
```
im = im/255
im -= [0.485, 0.456, 0.406]
im = im/[0.229, 0.224, 0.225]
```
